pages/api/cities/[city].js

```diff
@@ -71,6 +79,7 @@ handler.get(async (req, res) => {
     latitude,
     longitude,
     aqi: resp.data.aqi,
+    url: resp.data.city.url,
```

Comment:
I don't think we should be sending this back to the client, since this URL contains our API key which is sensitive data

Reply:
I would say we can just log it to the console so that we can see it for debugging purposes